hdat/store.py

```diff
@@ -20,9 +20,14 @@ def select(self, suite_id, case_id, result_id):
         else:
             return self.read_result(result_filename)
 
-    def select_recent(self, i, *args):
+    def select_recent(self, suites, i, *args):
         top_directory = os.path.join(self.root, *args)
```

Comment:
Short docstrings for the methods in `Archive` would be helpful as a reminder on what all these methods do, as well as showing an example of the *args format.

Reply:
Agreed